x-pack/plugins/lens/public/visualizations/xy/xy_config_panel/visual_options_popover/missing_values_option.tsx

```diff
@@ -88,6 +89,9 @@ export const MissingValuesOptions: React.FC<MissingValuesOptionProps> = ({
           {fittingFunction && fittingFunction !== FittingFunctions.NONE && (
             <>
               <EuiFormRow
+                css={css`
+                  max-inline-size: none !important;
+                `}
```

Comment:
Hi @Ajay-Satish-01 thank you for this solution. I think instead of using custom css we could take advantage of the `fullWidth` prop that eui exposes.

```suggestion
          fullWidth
```

Reply:
Ah. Got it. Should I commit the suggestion?